### ISSUE TITLE
Enhancement: Differential usage

### ISSUE THREAD

**Issue by dominikburri** [zavolanlab/PAQR2#8]
PAQR performs quantification of PAS usage based on the weighted average exon length. 
It is rather straightforward to test for differential PAS usage as the conditions are anyway supplied in the sample table. One could use the unpaired two-sample t-test or the Wilcoxon rank sum test to test for differential PAS usage between two conditions. One would need to check which test is more appropriate. In addition, p-value adjustment for multiple testing could be added, e.g. the Benjamini-Hochberg procedure to control the False Discovery Rate. 
Then the main output of PAQR would still be the average exon lengths but would additionally include the information which exons, resp. genes, show differential usage between conditions.

These procedures could be added as individual Snakemake rules after `weighted_avg_exon_lengths.tsv` is created. 

If the extension of PAQR is not desired, a note about this possibility and some pointers to how to do it, would increase the impact of PAQR.

**Comment by grexor** [comment]
One idea would be simply to have another comparisons.tab file, defining the comparisons (control set of samples vs test set of samples) and then producing the diff-usage file (one per comparison).

**Comment by AngryMaciek** [comment]
I am not maintaining this repository anymore.
We use PAQR2 as a module in MAPP: https://github.com/gruber-sciencelab/MAPP which I will expand in the forseable future.
There I will soon add some standalone scripts for various post-processing.
Would you mind opening an issue there and just link this one, please?

**Comment by uniqueg** [comment]
So who _is_ maintaining PAQR? Any idea, @mzavolan? Seems to me that the people asking for pointers on how to do downstream analysis have a good point, and this repo (and not MAPP) seems to be the right place to document that.

**Comment by mzavolan** [comment]
Yeah, it is the case that PAQR was taken into MAPP. I will have to look into whether we can implement the suggested feature.
